src/Tribe/Tickets.php

```diff
@@ -471,13 +471,35 @@ public function has_permission( $post, $data, $nonce_action ) {
 			}
 
 			if ( ! empty( $data['ticket_start_date'] ) ) {
-				$meridian           = ! empty( $data['ticket_start_meridian'] ) ? ' ' . $data['ticket_start_meridian'] : '';
-				$ticket->start_date = date( Tribe__Date_Utils::DBDATETIMEFORMAT, strtotime( $data['ticket_start_date'] . ' ' . $data['ticket_start_hour'] . ':' . $data['ticket_start_minute'] . ':00' . $meridian ) );
+				$start_hour     = ! empty( $data['ticket_start_hour'] ) ? $data['ticket_start_hour'] : '00';
```

Comment:
@bordoni I used exactly that, but my last commit undid it because our new code review bot complains about the dependency. 

I'm green with dependency injection, so maybe I'm wrong about the following. My thinking was, since this is an abstract class I would need to put each of the children in a container. Which is a bit beyond the scope of this little ticket IMO. Unless, you can think of an easier way to account for this?